src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -280,6 +292,21 @@ private void logIllegalBuildPublishOutput(Log log, ByteArrayOutputStream taskOut
         log.warn("Illegal build-publish output: " + taskOutputStream.toString(StandardCharsets.UTF_8));
     }
 
+    /**
+     * initialize values to be used across the class.
+     * @param env       environment variables applicable to this step
+     * @param launcher  a way to start processes
+     * @param workspace a workspace to use for any file operations
+     * @throws IOException in case of any I/O error, or we failed to run the 'jf'
+     * @throws InterruptedException if the step is interrupted
```

Comment:
(Please use IntelliJ formatter to reformat the code)
```suggestion
     * Initializes values to be used across the class.
     *
     * @param env       environment variables applicable to this step
     * @param launcher  a way to start processes
     * @param workspace a workspace to use for any file operations
     * @throws IOException          in case of any I/O error, or we failed to run the 'jf'
     * @throws InterruptedException if the step is interrupted
```